### ISSUE TITLE
Can you add a feature to disable the required fields

### ISSUE THREAD

**Issue by ellemaker** [marcbelletre/acf-rrule#10]

**Comment by marcbelletre** [comment]
What do you mean? The only required field is the start date. We can't make it optional otherwise the plugin can't work.

**Comment by ellemaker** [comment]
yes, because I have an event that will not have recurring dates. can it be done if the start date is set up then the plugin will work?

**Comment by marcbelletre** [comment]
Oh okay I see. I'll have a look at it tomorrow. In the meantime you can add conditional logic on your RRule field based on a true/false field. This will bypass the validation.

**Comment by ellemaker** [comment]
I already done this but it won't submit or publish the post

**Comment by marcbelletre** [comment]
I just tested out and it can confirm that it works. What version of ACF and ACF RRule are you using?

Here the validation fails when I check the box and leave the field empty:
<img width="1121" alt="Capture d’écran 2021-08-04 à 09 15 20" src="https://user-images.githubusercontent.com/6853334/128138149-e2c50b52-a15a-49b3-9adf-4ca2e90f5e74.png">

But it works when I uncheck it:
<img width="1121" alt="Capture d’écran 2021-08-04 à 09 15 27" src="https://user-images.githubusercontent.com/6853334/128138160-3cf0cf74-e7d2-4c06-bc4f-7a8d61d12d06.png">

**Comment by ellemaker** [comment]
ACF PRo : Version 5.9.9 
RRule : Version 1.2.1

**Comment by ellemaker** [comment]
Try it on custom post type without a content/editor field

**Comment by marcbelletre** [comment]
Hi @ellemaker,

I just released a new version with improved validation. This includes the possibility to disable required fields as requested. If you don't make RRule field as required then nothing will be saved unless you specify a start date.

**Comment by ellemaker** [comment]
@marcbelletre thank you 🥳🥳🥳
